code/modules/antagonists/bloodsucker/objects/bloodsucker_crypt.dm

```diff
@@ -486,9 +484,7 @@
 	if(lit)
 		for(var/mob/living/carbon/human/H in viewers(7, src))
 			var/datum/antagonist/vassal/T = H.mind.has_antag_datum(ANTAG_DATUM_VASSAL)
-			var/datum/antagonist/bloodsucker/V = H.mind.has_antag_datum(ANTAG_DATUM_BLOODSUCKER)
-			if(V || T) //We dont want vassals or vampires affected by this
-				return
+			if(!isvamp(H) || !T) //We dont want vassals or vampires affected by this
```

Comment:
and not or? this is where travis failed

Reply:
Whoops, will fix tommorow

Reply:
It compiles locally, goddamnit it seems i lost some of the changes